tests/5.1/error/test_error_message.c

```diff
@@ -23,7 +23,7 @@ int test_error_message() {
     {
         #pragma omp single
         {
-            #pragma omp error message("error message success")
+            #pragma omp error severity(warning) message("error message success")
```

Comment:
Agree, since default is "fatal".

Reply:
@fel-cab : I don't disagree, but why does it matter when the msg is generated for this test ?

Reply:
I understand that the test checks the error message at execution, so I don't see the need to have a warning at compilation. Furthermore, it could be a problem if it is compiled with a flag to treat warnings as errors.